test/unit_tests/arkode/CXX_serial/CMakeLists.txt

```diff
@@ -54,7 +54,20 @@ set(unit_tests
     "ark_test_slowerror_brusselator.cpp\;\;exclude-single"
     "ark_test_slowerror_kpr.cpp\;\;exclude-single"
     "ark_test_slowerror_polynomial.cpp\;\;exclude-single"
-    "ark_test_splittingstep.cpp\;\;")
+    "ark_test_splittingstep.cpp\;\;"
+    "ark_test_adjoint_erk.cpp\;--check-freq 1\;"
+    "ark_test_adjoint_erk.cpp\;--check-freq 2\;"
+    "ark_test_adjoint_erk.cpp\;--check-freq 5\;"
+    "ark_test_adjoint_erk.cpp\;--check-freq 1 --dont-keep\;"
+    "ark_test_adjoint_erk.cpp\;--check-freq 2 --dont-keep\;"
+    "ark_test_adjoint_erk.cpp\;--check-freq 5 --dont-keep\;"
+    "ark_test_adjoint_ark.cpp\;--check-freq 1\;"
+    "ark_test_adjoint_ark.cpp\;--check-freq 2\;"
+    "ark_test_adjoint_ark.cpp\;--check-freq 5\;"
+    "ark_test_adjoint_ark.cpp\;--check-freq 1 --dont-keep\;"
+    "ark_test_adjoint_ark.cpp\;--check-freq 2 --dont-keep\;"
+    "ark_test_adjoint_ark.cpp\;--check-freq 5 --dont-keep\;"
```

Comment:
Add tests with `--no-stages` and output files are missing for the `--dont-keep` cases

Reply:
I removed the save_stages option as its not yet supported.

Reply:
Do you want to keep the input flag to the constructor but not that it is not currently supported so that adding support does not require an API change (or a new constructor)?

Reply:
We discussed offline and this option can be supported in future by adding "Set" function